### ISSUE TITLE
ADA Color Contrast Violation

### ISSUE THREAD

**Issue by ali-kashanian** [paypal/paypal-messaging-components#931]
Hello,

During my test, I found an exclamation mark that didn't have sufficient contrast (WCAG 2.1 1.4.11 Non-text Contrast (Level AA)):

![image](https://user-images.githubusercontent.com/16331776/235573444-c7d14bdf-015a-472b-8dcd-b4694247c3e7.png)

The exclamation mark is an SVG with the color #FF9600, and has 2.184:1 contrast ratio against the white background (#FFFFFF), whereas it should be a bit darker to have at least 3:1 contrast ratio.

``` HTML
<svg aria-hidden="true" width="18" height="18" viewBox="0 0 18 18" fill="none" xmlns="http://www.w3.org/2000/svg"><path fill-rule="evenodd" clip-rule="evenodd" d="M10.3074 1.80394L17.5975 14.7642C17.7975 15.1197 17.8369 15.5218 17.7393 15.887C17.6386 16.2635 17.3921 16.6008 17.0255 16.807C16.801 16.9333 16.5478 16.9996 16.2902 16.9996H1.70985C0.881419 16.9996 0.209846 16.328 0.209846 15.4996C0.209846 15.242 0.276189 14.9887 0.402483 14.7642L7.69264 1.80394C7.89219 1.44918 8.21448 1.2069 8.57637 1.10039C8.95102 0.990125 9.36811 1.02537 9.73539 1.23197C9.97483 1.36665 10.1727 1.5645 10.3074 1.80394ZM9 4.9996C8.44771 4.9996 8 5.44731 8 5.9996V9.9996C8 10.5519 8.44771 10.9996 9 10.9996C9.55228 10.9996 10 10.5519 10 9.9996V5.9996C10 5.44731 9.55228 4.9996 9 4.9996ZM10 13.9996C10 14.5519 9.55228 14.9996 9 14.9996C8.44771 14.9996 8 14.5519 8 13.9996C8 13.4473 8.44771 12.9996 9 12.9996C9.55228 12.9996 10 13.4473 10 13.9996Z" fill="#FF9600"></path></svg>
```

Please let me know if you needed more information to troubleshoot this issue.

Kind regard.

**Comment by erikacolette29** [comment]
Update: We have pushed out the fix. PR: [https://github.com/paypal/paypal-messaging-components/pull/933](url)
